SharedSource/Core/ReSharperAnnotations/JetbrainsAnnotations.cs

```diff
@@ -54,7 +57,8 @@ namespace JetBrains.Annotations
     AttributeTargets.Method | AttributeTargets.Parameter | AttributeTargets.Property |
     AttributeTargets.Delegate | AttributeTargets.Field | AttributeTargets.Event |
     AttributeTargets.Class | AttributeTargets.Interface | AttributeTargets.GenericParameter)]
-internal sealed class CanBeNullAttribute : Attribute { }
+  [Conditional("JETBRAINS_ANNOTATIONS")]
```

Comment:
please check, I believe you have used "internal+conditonal" instead of only "conditonal".

Please redo.